identity/src/main/java/com/shotcutter/identity/user/UserIdentityService.java

```diff
@@ -93,15 +93,15 @@ public UserIdentityService(@Value("${s3.bucketName}") String s3BucketName,
 
     private Mono<DeleteObjectResponse> deleteUserAvatar(UserEntity user) {
         if (user.getAvatar() == null || !user.getAvatar().contains(s3BucketName)) {
-            return Mono.empty();
+            return Mono.just(null);
         }
 
         String avatarPath;
 
         try {
             avatarPath = new URL(user.getAvatar()).getPath();
         } catch (MalformedURLException e) {
-            return Mono.empty();
+            return Mono.just(null);
```

Comment:
empty mono?

Reply:
again, looks like you missed rebase

Reply:
`Mono#empty` doesn't resolves, so yep, this one make sense